lib/ksmbdtools.c

```diff
@@ -14,6 +14,9 @@
 #include <stdio.h>
 #include <ksmbdtools.h>
 
+int log_level = PR_INFO;
```

Comment:
Is there any reason that the variable, log_level is moved? All utilities share the log level.

Reply:
@hclee, `log_level` was static. Each source file that included `ksmbdtools.h` had its own copy.

Reply:
Does this mean that the current implementation doesn't change the log level of other running utils? e.g. ksmbd.mountd is running with default option and we call ksmbd.adduser with -v option.

Reply:
@namjaejeon, before these changes, each source file that included `ksmbdtools.h` had its own `log_level`. After this change, each utility has its own `log_level`. I don't understand how `log_level` of one utility could possibly affect `log_level` of other utilities.

Reply:
Ah, I thought libksmbdtools is share library. Okay. understood.